go.mod

```diff
@@ -3,12 +3,18 @@ module sigs.k8s.io/kubebuilder/v3
 go 1.15
 
 require (
+	github.com/cloudflare/cfssl v1.5.0
 	github.com/gobuffalo/flect v0.2.2
+	github.com/markbates/pkger v0.17.1
 	github.com/onsi/ginkgo v1.12.0
 	github.com/onsi/gomega v1.9.0
 	github.com/spf13/afero v1.2.2
-	github.com/spf13/cobra v0.0.7
+	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/tools v0.0.0-20200403190813-44a64ad78b9b
+	golang.org/x/tools v0.0.0-20190920225731-5eefd052ad72
+	k8s.io/api v0.20.2 // indirect
+	k8s.io/apimachinery v0.20.2
+	sigs.k8s.io/controller-tools v0.3.0
+	sigs.k8s.io/kustomize/kyaml v0.10.10
```

Comment:
IMO: 
- config-gen does not need controller-tools. If we write the file on disk it ought to use the version defined in the Makefile which is not the same for all plugin/go versions. And then, if we want to use it without write in the disk then I think it needs to be configurable in its file instead.